### ISSUE TITLE
docker image 9.0.0 is not working

### ISSUE THREAD

**Issue by bobelev** [bacula-web/bacula-web#180]
Current Docker container uses PHP 8.0, but some dependencies require at least 8.1

```
:~/# docker run -ti --rm --entrypoint=""  baculaweb/bacula-web:latest php  --version
PHP 8.0.30 (cli) (built: Oct 12 2023 05:07:52) ( NTS )
Copyright (c) The PHP Group
Zend Engine v4.0.30, Copyright (c) Zend Technologies

:~/# docker run -ti --rm --entrypoint=""  baculaweb/bacula-web:latest php  /var/www/html/vendor/symfony/twig-bridge/Extension/TranslationExtension.php
PHP Parse error:  syntax error, unexpected token ")" in /var/www/html/vendor/symfony/twig-bridge/Extension/TranslationExtension.php on line 64
```

Links:
- https://github.com/symfony/symfony/blame/6.4/src/Symfony/Bridge/Twig/Extension/TranslationExtension.php#L64
- https://www.php.net/manual/en/functions.first_class_callable_syntax.php

Also, It would be nice to have `Dockerfile` inside this project to tinker with the image.

**Comment by dfranco** [comment]
> Current Docker container uses PHP 8.0, but some dependencies require at least 8.1
> 
> ```
> :~/# docker run -ti --rm --entrypoint=""  baculaweb/bacula-web:latest php  --version
> PHP 8.0.30 (cli) (built: Oct 12 2023 05:07:52) ( NTS )
> Copyright (c) The PHP Group
> Zend Engine v4.0.30, Copyright (c) Zend Technologies
> 
> :~/# docker run -ti --rm --entrypoint=""  baculaweb/bacula-web:latest php  /var/www/html/vendor/symfony/twig-bridge/Extension/TranslationExtension.php
> PHP Parse error:  syntax error, unexpected token ")" in /var/www/html/vendor/symfony/twig-bridge/Extension/TranslationExtension.php on line 64
> ```
> 
> Links:
> 
> * https://github.com/symfony/symfony/blame/6.4/src/Symfony/Bridge/Twig/Extension/TranslationExtension.php#L64
> * https://www.php.net/manual/en/functions.first_class_callable_syntax.php
> 

Hi @bobelev 

Thanks for the bug report

I just made a test on my side ....

```
docker run --rm -it -u www-data baculaweb/bacula-web:latest php bwc check

=====================
Checking requirements
=====================

	PHP version Ok (using 8.0.30)
	PHP timezone Warning
	Protected assets folder is writable Error
	Twig cache folder write permission Ok

Checking installed PHP database extensions
==========================================

	sqlite installed
	mysql installed
	pgsql installed

Checking installed PHP extensions
=================================

	SQLite support Ok
	Gettext support Ok
	Session support Ok
	PHP PDO support Ok
	PHP Posix support Ok
```

```
docker run --rm --name bacula-web \
  -d -p 8000:80 \
  -v $PWD/config.php:/var/www/html/application/config/config.php \
  -v $PWD/protected:/var/www/html/application/assets/protected \
  baculaweb/bacula-web:latest

CONTAINER ID   IMAGE                           COMMAND                  CREATED          STATUS          PORTS                                   NAMES
13a1bd513b62   baculaweb/bacula-web:latest     "docker-php-entrypoi…"   13 seconds ago   Up 13 seconds   0.0.0.0:8000->80/tcp, :::8000->80/tcp   bacula-web
```

when I try to connect to http://localhost:8000, I see the same error in the log

`[Thu Oct 26 19:00:57.170725 2023] [php:error] [pid 20] [client 172.17.0.1:56722] PHP Parse error:  syntax error, unexpected token ")" in /var/www/html/vendor/symfony/twig-bridge/Extension/TranslationExtension.php on line 64`

I'm quiet sure it's not related to PHP version as my dev env is using PHP 8.0.

I'd need some time to investigate, I'll keep you posted

> Also, It would be nice to have `Dockerfile` inside this project to tinker with the image.

good point, I'll publish it soon

**Comment by bobelev** [comment]
I've tried to run `TranslationExtension.php` with 8.1 interpreter. It doesn't raise any errors.

Maybe somehow you have 8.1 as your dev version?

Specifically this line was committed with the message "Bump minimum version of PHP to 8.1" —  https://github.com/symfony/symfony/commit/b0217c600f078d58d6c3bd9895d6a015d1f24c14#diff-8570531dce0b9cafa3ebf3c58860f2ad62eb10fc2710dff6a8fac96e4448c213R67

**Comment by dfranco** [comment]
> I've tried to run `TranslationExtension.php` with 8.1 interpreter. It doesn't raise any errors.
> 
> Maybe somehow you have 8.1 as your dev version?
 no I'm sure, here's the proof ;)

```
docker-compose exec -it -u www-data web php -v
PHP 8.0.28 (cli) (built: May  3 2023 05:34:55) ( NTS )
Copyright (c) The PHP Group
Zend Engine v4.0.28, Copyright (c) Zend Technologies
    with Xdebug v3.1.5, Copyright (c) 2002-2022, by Derick Rethans
```

> 
> Specifically this line was committed with the message "Bump minimum version of PHP to 8.1" — [symfony/symfony@b0217c6#diff-8570531dce0b9cafa3ebf3c58860f2ad62eb10fc2710dff6a8fac96e4448c213R67](https://github.com/symfony/symfony/commit/b0217c600f078d58d6c3bd9895d6a015d1f24c14#diff-8570531dce0b9cafa3ebf3c58860f2ad62eb10fc2710dff6a8fac96e4448c213R67)

interesting, I'll have a deeper look, thanks @bobelev

**Comment by dfranco** [comment]
@bobelev I manage to find a fix and will publish the Docker images in a moment

Thanks a lot again for you help

**Comment by dfranco** [comment]
Fixed Docker image published on Docker Hub, issue #183 created to push Docker resources.
